documentation/modules/ROOT/partials/modules/all-connectors/con-connector-incremental-snapshot.adoc

```diff
@@ -0,0 +1,177 @@
+ifdef::community[]
+Incremental snapshots are based on the link:https://github.com/debezium/debezium-design-documents/blob/main/DDD-3.md[DDD-3] design document.
+endif::community[]
+
+In an incremental snapshot, instead of capturing the full state of a database all at once, as in an initial snapshot, {prodname} captures each {data-collection} in phases, in a series of configurable chunks.
+You can specify the {data-collection}s that you want the snapshot to capture and the xref:{context}-property-incremental-snapshot-chunk-size[size of each chunk].
+The chunk size determines the number of rows that the snapshot collects during each fetch operation on the database.
+The default chunk size for incremental snapshots is 1 KB.
+
+As an incremental snapshot proceeds, {prodname} uses watermarks to track its progress, maintaining a record of each {data-collection} row that it captures.
+This phased approach to capturing data provides the following advantages over the standard initial snapshot process:
+
+* You can run incremental snapshots in parallel with streamed data capture, instead of postponing streaming until the snapshot completes.
+  The connector continues to capture near real-time events from the change log throughout the snapshot process, and neither operation blocks the other.
+* If the progress of an incremental snapshot is interrupted, you can resume it without losing any data.
+  After the process resumes, the snapshot begins at the point where it stopped, rather than recapturing the {data-collection} from the beginning.
+* You can run an incremental snapshot on demand at any time, and repeat the process as needed to adapt to database updates.
+  For example, you might re-run a snapshot after you modify the connector configuration to add a {data-collection} to its xref:{context}-property-table-include-list[`table.include.list`] property.
+
+.Incremental snapshot process
+When you run an incremental snapshot, {prodname} sorts each {data-collection} by primary key and then splits the {data-collection} into chunks based on the xref:{context}-property-incremental-snapshot-chunk-size[configured chunk size].
+Working chunk by chunk, it then captures each {data-collection} row in a chunk.
+For each row that it captures, the snapshot emits a `READ` event.
+That event represents the value of the row when the snapshot for the chunk began.
+
+As a snapshot proceeds, it’s likely that other processes continue to access the database, potentially modifying {data-collection} records.
+To reflect such changes, `INSERT`, `UPDATE`, or `DELETE` operations are committed to the transaction log as per usual.
+Similarly, the ongoing {prodname} streaming process continues to detect these change events and emits corresponding change event records to Kafka.
+
+.How {prodname} resolves conflicts among records with the same primary key
```

Comment:
I prefere the word collision but it is ok to use conflict too. Why it was chosen?

Reply:
Thanks. Yes, collision is a better choice, but I didn't think of it.  I'll make the change.